test/test.js

```diff
@@ -29,6 +29,7 @@ const iPhoneLandscape = DeviceDescriptors['iPhone 6 landscape'];
 
 const SimpleServer = require('./server/SimpleServer');
 const GoldenUtils = require('./golden-utils');
+const FrameUtils = require('./frame-utils');
```

Comment:
I didn't find a reason no to require the util once on the top.

Reply:
this is good, thanks